src/e2e-test/java/io/cdap/plugin/utils/BigTableClient.java

```diff
@@ -0,0 +1,100 @@
+import com.google.cloud.bigtable.admin.v2.models.CreateInstanceRequest;
+import com.google.cloud.bigtable.admin.v2.models.Instance;
+import com.google.cloud.bigtable.admin.v2.models.StorageType;
+import com.google.cloud.bigtable.hbase.BigtableConfiguration;
+import com.google.cloud.bigtable.hbase.BigtableOptionsFactory;
+import com.google.common.collect.ImmutableList;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.HColumnDescriptor;
+import org.apache.hadoop.hbase.HTableDescriptor;
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.client.Connection;
+import org.apache.hadoop.hbase.client.Put;
+import org.apache.hadoop.hbase.client.Table;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.codehaus.commons.nullanalysis.NotNull;
+
+import java.io.IOException;
+import java.util.List;
+import javax.annotation.Nullable;
+
+
+/**
+ * Represents GCP CBT Client.
+ */
+public class BigTableClient {
+    public static void createBigTableInstance(BigtableInstanceAdminClient adminClient,
+                                              String instanceId, String clusterId) {
+        if (!adminClient.exists(instanceId)) {
+            CreateInstanceRequest createInstanceRequest =
+                    CreateInstanceRequest.of(instanceId)
+                            .addCluster(clusterId, "us-central1-f",
+                                    3, StorageType.SSD)
+                            .setType(Instance.Type.PRODUCTION)
+                            .addLabel("it", "e2e-testing");
+            try {
+                Instance instance = adminClient.createInstance(createInstanceRequest);
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+    }
+    public static void deleteInstance(BigtableInstanceAdminClient adminClient, String bigTableInstanceId) {
+        try {
+            adminClient.deleteInstance(bigTableInstanceId);
+        } catch (NotFoundException e) {
+            e.printStackTrace();
+        }
+    }
+    public static Connection connect(String projectId, String instanceId, @Nullable String serviceAccountCredentials) {
+        Configuration configuration = BigtableConfiguration.configure(projectId, instanceId);
+        if (serviceAccountCredentials != null) {
+            configuration.set(BigtableOptionsFactory.BIGTABLE_SERVICE_ACCOUNT_JSON_VALUE_KEY,
+                    serviceAccountCredentials);
+        }
+        return BigtableConfiguration.connect(configuration);
+    }
+    public static void createTables(Connection connection,
+                                    @Nullable String sourceTable, @Nullable String sinkTable) throws IOException {
+
+        List<String> families = ImmutableList.of("cf1", "cf2");
+        if (sourceTable != null) {
+            createTable(connection,
+                    sourceTable, families);
+        }
+        if (sinkTable != null) {
+            createTable(connection,
```

Comment:
NIT : no need of indentation, keep in single line

Reply:
done